netconnection.go

```diff
@@ -29,6 +40,14 @@ func (nc *netConnection) ConnectionID() string {
 	return nc.connectionID
 }
 
+func (nc *netConnection) SetConnectionID(id string) {
```

Comment:
these methods were added long after netConnection was used the last time in the chatsample